python/cudf/cudf/_libxx/scalar.pxd

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
Should this file go in `includes` instead of just in `_libxx`?

Reply:
This is similar to aggregation.pxd, and table.pxd. If moved to includes, it errors.

Reply:
How does it error?

Reply:
It will not be able to find it.

Reply:
Don't we just need to change the import path to have `.includes` then?

Reply:
Needed a clean build, I have removed the scalar.pxd, it is not required anymore.